sdk/identity/Azure.Identity/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,4 @@
+﻿using Azure.Core;
+using System.Runtime.CompilerServices;
+
+[assembly:AzureSdkClientLibrary("identity")]
```

Comment:
Should this be fill name? @pakrym? Could we add a static analysis rule to make sure people pass correct string format here?

Reply:
What would be "correct" here, it's just a value that's gets sent with User-Agent header.  There is https://github.com/Azure/azure-sdk-for-net/issues/5618 to define how to go from the long name to short one.